app/controllers/comments/votes_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Comments::VotesController < ApplicationController
+  before_action :ensure_current_user
+  before_action :set_comment
+
+  def create
+    current_user.likes @comment
+    insert_comment @comment
+
+    redirect_to :back
+  end
+
+  def destroy
+    current_user.unlike @comment
+    insert_comment @comment
+
+    redirect_to :back
+  end
+
+  private
+    def votable_params
+      params.permit(:id)
+    end
+
+    def set_comment
+      @comment = Comment.find(votable_params[:id])
+    end
+
+    def insert_comment comment
+      if comment.parent_id
+        closest_comment_by_votes = comment.siblings.min_by {|c| (c.cached_votes_up - comment.cached_votes_up).abs }
+        if comment.cached_votes_up > closest_comment_by_votes.cached_votes_up
+          closest_comment_by_votes.prepend_sibling(comment)
+        elsif comment.cached_votes_up == closest_comment_by_votes.cached_votes_up
+          newest_similar_comment = comment.siblings.where(
+            cached_votes_up: closest_comment_by_votes.cached_votes_up).order(updated_at: :asc).last
+          newest_similar_comment.append_sibling(comment)
```

Comment:
Woops, I was trying to push to a new branch to submit a PR and git wasn't having it.

```
briankung:projectcredo (votesForComments)$ gp -u origin appendCommentSort
error: src refspec appendCommentSort does not match any.
error: failed to push some refs to 'git@github.com:briankung/projectcredo.git'
Counting objects: 12, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (10/10), done.
Writing objects: 100% (12/12), 1.12 KiB | 0 bytes/s, done.
Total 12 (delta 8), reused 0 (delta 0)
remote: Resolving deltas: 100% (8/8), completed with 4 local objects.
To github.com:briankung/projectcredo.git
   ff4fd3b..c178d76  votesForComments -> votesForComments
Branch votesForComments set up to track remote branch votesForComments from origin.
```